Source/Core/VideoBackends/D3DCommon/SwapChain.cpp

```diff
@@ -164,12 +203,19 @@ bool SwapChain::ResizeSwapChain()
 {
   DestroySwapChainBuffers();
 
-  HRESULT hr = m_swap_chain->ResizeBuffers(SWAP_CHAIN_BUFFER_COUNT, 0, 0,
-                                           GetDXGIFormatForAbstractFormat(m_texture_format, false),
+  // The swap chain fills up the size of the window if no size is specified
+  HRESULT hr = m_swap_chain->ResizeBuffers(SWAP_CHAIN_BUFFER_COUNT, 0, 0, DXGI_FORMAT_UNKNOWN,
                                            GetSwapChainFlags());
+
   if (FAILED(hr))
     WARN_LOG_FMT(VIDEO, "ResizeBuffers() failed: {}", Common::HRWrap(hr));
 
+  Microsoft::WRL::ComPtr<IDXGISwapChain4> swap_chain4;
+  hr = m_swap_chain->QueryInterface(IID_PPV_ARGS(&swap_chain4));
+  if (SUCCEEDED(hr))
+    hr = swap_chain4->SetColorSpace1(m_hdr ? DXGI_COLOR_SPACE_RGB_FULL_G10_NONE_P709 :
```

Comment:
Should we be setting this if `m_hdr` is not true?  I'm sure it doesn't hurt but  I do not see `DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709` anywhere else in our code and `SetColorSpace1` was not set before.

Reply:
It's just there for consistency and clarity. It sets the default value anyway. I think we can leave it.

Reply:
One other thing, multi-line statements need to be wrapped in `{` `}` if in an `if` statement